dti-website-redesign/src/components/Footer.module.css

```
@@ -24,4 +24,38 @@
 .socialIcon {
   height: 35px;
   width: 35px;
+  cursor: pointer;
+}
+
+.emailCopyNotification {
+  animation: fadeAndBounce 4s 1;
```

I'm sorry, i was wrong before. When i removed the translate functions locally it still worked, but I see in the deploy preview that its wrong the first time you click the icon, but right every time you click afterwards?? Im not sure why that is, but I think its better to go back to manually adjusting the notification position so that it works every time. You can still reduce duplication tho if you adjust the position of the notification itself instead of during the animation:
```suggestion
  position: absolute;
  left: -105px;
  top: -50px;
  animation: fadeAndBounce 4s 1;
```

---

No worries! Yeah I saw something earlier which is why I think I did that position absolute thing...idk if it helps but I found initially that when the notification is in the middle, and you resize the window, the notification actually snaps into place. Maybe it has to do with its position being "flex". Anyways, made the change.